lib/extensions/reset_embroidery_settings.py

```diff
@@ -0,0 +1,47 @@
+import inkex
+
+from .base import InkstitchExtension
+
+
+class ResetEmbroiderySettings(InkstitchExtension):
+    def effect(self):
+        # TODO: Extension does not work for ignored layers/objects
+        if not self.get_elements():
+            return
+
+        for element in self.elements:
+            node = element.node
+            for attrib in node.attrib:
+                if self.selected:
+                    xpath = (".//svg:path[@inkscape:connection-start='#%(id)s' or @inkscape:connection-end='#%(id)s']/parent::*" %
+                             dict(id=node.get('id')))
+                    commands = self.remove_elements(xpath)
+                if attrib.startswith('embroider_'):
+                    del node.attrib[attrib]
+
+        # Delete all commands and print information if no object is selected
```

Comment:
While we're at it, we should get rid of any layer and global commands too, probably.

Reply:
Hmm, if no object is selected, we should probably remove _all_ `embroider_` attributes from everything, too?